components/telnyx/actions/send-message/send-message.mjs

```diff
@@ -0,0 +1,99 @@
+import telnyxApp from "../../telnyx.app.mjs";
+
+export default {
+    key: "telnyx-send-message",
+    name: "Send Message",
+    description: "Send an SMS or MMS message. See documentation [here](https://developers.telnyx.com/docs/messaging/messages/send-message)",
+    version: "1.0.0",
+    type: "action",
+    props: {
+        telnyxApp,
+        messaging_profile_id: {
+            optional: true,
+            propDefinition: [
+                telnyxApp,
+                "messaging_profile_id",
+            ],
+        },
+        to: {
+            type: "string",
+            label: "To",
+            description: "Receiving address (+E.164 formatted phone number or short code).",
+        },
+        from: {
+            type: "string",
+            label: "From",
+            description: "Sending address (+E.164 formatted phone number or short code). Required if sending with a phone number, short code, or alphanumeric sender ID. See [Sending a message on the Telnyx platform](https://developers.telnyx.com/docs/messaging/messages/send-message) for more details.",
+            optional: true,
+        },
+        text: {
+            type: "string",
+            label: "Text",
+            description: "Message content. Must be a valid UTF-8 string, and no longer than 1600 characters for SMS or 5MB for MMS. Required if sending an SMS message.",
+            optional: true,
+        },
+        subject: {
+            type: "string",
+            label: "Subject",
+            description: "Subject of the MMS message.",
+            optional: true,
+        },
+        media_urls: {
```

Comment:
These props should follow the camelCase naming convention, e.g. `mediaUrls`, `webhookUrl` etc.

Reply:
Resolved in commit 4fc67138a.